EEPROM/README.md

```diff
@@ -1,49 +1,47 @@
 # Utilities to create, flash and dump HAT EEPROM images.
 
+Original source of the scripts and templates:
+https://github.com/raspberrypi/hats/tree/master/eepromutils
+
 There is a complete, worked example at:
 https://www.raspberrypi.org/forums/viewtopic.php?t=108134
 
 ## Usage
 
```

Comment:
Can you add a note that this is run on the Pi.

Reply:
Yes